networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobSidecarsByRangeListenerValidatingProxyTest.java

```diff
@@ -214,35 +206,23 @@ void blobSidecarIndexIsGreaterOrEqualThanMaxBlobs() {
         new BlobSidecarsByRangeListenerValidatingProxy(
             spec, peer, listener, maxBlobsPerBlock, kzg, startSlot, count);
 
-    final Bytes32 blockRoot1 = dataStructureUtil.randomBytes32();
-    final BlobSidecarOld blobSidecar1 =
-        dataStructureUtil.randomBlobSidecarOld(UInt64.ONE, blockRoot1, Bytes32.ZERO, UInt64.ZERO);
-    final BlobSidecarOld blobSidecar2 =
-        dataStructureUtil.randomBlobSidecarOld(UInt64.ONE, blockRoot1, Bytes32.ZERO, UInt64.ONE);
-    final BlobSidecarOld blobSidecar3 =
-        dataStructureUtil.randomBlobSidecarOld(
-            UInt64.ONE, blockRoot1, Bytes32.ZERO, UInt64.valueOf(2));
-    final BlobSidecarOld blobSidecar4 =
-        dataStructureUtil.randomBlobSidecarOld(
-            UInt64.ONE, blockRoot1, Bytes32.ZERO, UInt64.valueOf(3));
-    final BlobSidecarOld blobSidecar5 =
-        dataStructureUtil.randomBlobSidecarOld(
-            UInt64.ONE, blockRoot1, Bytes32.ZERO, UInt64.valueOf(4));
-    final BlobSidecarOld blobSidecar6 =
-        dataStructureUtil.randomBlobSidecarOld(
-            UInt64.ONE, blockRoot1, Bytes32.ZERO, UInt64.valueOf(5));
-    final BlobSidecarOld blobSidecar7 =
-        dataStructureUtil.randomBlobSidecarOld(
-            UInt64.ONE, blockRoot1, Bytes32.ZERO, UInt64.valueOf(6));
-
-    safeJoin(listenerWrapper.onResponse(blobSidecar1));
-    safeJoin(listenerWrapper.onResponse(blobSidecar2));
-    safeJoin(listenerWrapper.onResponse(blobSidecar3));
-    safeJoin(listenerWrapper.onResponse(blobSidecar4));
-    safeJoin(listenerWrapper.onResponse(blobSidecar5));
-    safeJoin(listenerWrapper.onResponse(blobSidecar6));
-
-    final SafeFuture<?> result = listenerWrapper.onResponse(blobSidecar7);
+    final SignedBeaconBlock block1 = dataStructureUtil.randomSignedBeaconBlock(ONE);
+    final BlobSidecar blobSidecar10 = dataStructureUtil.randomBlobSidecarForBlock(block1, 0);
```

Comment:
My brain suffers with this numbering. what about `blobSidecar1_0` or `blobSidecar1i0` or something else. Here and the other tests too.